api/queries_release.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"time"
+)
+
+var ReleaseFields = []string{
+	"url",
+	"tarballUrl",
+	"zipballUrl",
+	"id",
+	"tagName",
+	"name",
+	"description",
+	"isDraft",
+	"isLatest",
+	"isPrerelease",
+	"createdAt",
+	"publishedAt",
+	"author",
+	"releaseAssets",
+}
+
+type Release struct {
+	DatabaseID   int64
+	ID           string
+	TagName      string
+	Name         string
+	Description  string
+	IsDraft      bool
+	IsPrerelease bool
+	CreatedAt    time.Time
+	PublishedAt  *time.Time
+	URL          string
+	ResourcePath string
+	Author       Author
+
+	ReleaseAssets struct {
+		TotalCount int
+		Nodes      []ReleaseAsset
+	} `graphql:"releaseAssets(first: 100)"`
+
+	Tag struct {
+		ID   string
+		Name string
+
+		Target struct {
+			OID string
+			// commitResourcePath string
```

Comment:
When uncommented, this fails with `struct field for "commitResourcePath" doesn't exist in any of 1 places to unmarshal`. However, copying the generated query to the GH Explorer and executing works without issue. Is there a versioning issue at play, perhaps?